Sources/Common/DataModel/PolyLine/index.js

```diff
@@ -0,0 +1,120 @@
+import macro from 'vtk.js/Sources/macros';
+import vtkCell from 'vtk.js/Sources/Common/DataModel/Cell';
+import vtkLine from 'vtk.js/Sources/Common/DataModel/Line';
+
+function vtkPolyLine(publicAPI, model) {
+  model.classHierarchy.push('vtkPolyLine');
+  const superClass = { ...publicAPI };
+
+  model.line = vtkLine.newInstance();
```

Comment:
no need to store in model, can be a private variable (`const line = vtkLine.newInstance()`).